decidim-proposals/app/helpers/decidim/proposals/admin/proposals_picker_helper.rb

```diff
@@ -1,10 +1,10 @@
 # frozen_string_literal: true
 
+# deprecated
```

Comment:
I would prefer to remove, either to add a ActiveSupport::Deprecation.warn("...") message.